src/command_modules/azure-cli-interactive/azclishell/configuration.py

```diff
@@ -140,3 +154,6 @@ def ask_user_for_telemetry():
 
 
 CONFIGURATION = Configuration()
+
+if not az_config.has_option('core', 'given feedback'):
+    set_global_config_value('core', 'given feedback', 'no')
```

Comment:
Could assume that if the config value isn't there, it isn't set instead of forcibly setting it to no.